contribs/src/main/java/com/netflix/conductor/contribs/http/HttpTask.java

```diff
@@ -54,13 +55,75 @@
 /**
  * @author Viren
  * Task that enables calling another http endpoint as part of its execution
+ * 
+ * Basic support is to fail responses < 200 and > 299
+ * If you desire additional support for any specific response code  
+ * set map input_Parameter:status_support, keys are httpStatuses and value is TaskStatus to be set
+ * Use a decision tree task after completion to handle specific httpStatuses
+ * input_Parameter:alternate_workflow can be used to indicated default workflow for httpStatuses that are invoked.
  */
+/**
+ * @author jstanley 
+ * 
+ * Task updateds to allow http statuses (>200 and > 299)
+ * to be handled with custom handlers.
+ * 
+ * Basic support remains the same to fail responses < 200 and > 299 (exactly the same
+ * as orginally designed) Setting http_status_override values overrides default
```

Comment:
Doesn't basic support fail responses **within** the range of 200-299 - that seems to be the opposite of what you're saying